openapi/components/schemas/Organizations/OrganizationSettings.yaml

```
@@ -25,3 +25,23 @@ properties:
       Available only for US-based organizations.
     items:
       $ref: ./TaxLocation.yaml
+  notifications:
+    type: object
+    description: Notification settings for the organization.
+    properties:
+      notifyOnUserAccessChanges:
+        description: Specifies if a notification should be sent when there are changes in user access.
+        type: boolean
+        default: false
+        example: false
+      notifyOnApiKeyAccessChanges:
+        description: Specifies if a notification should be sent when there are changes in API key access.
+        type: boolean
+        default: false
+        example: false
+      notificationEmailAddresses:
```

```suggestion
      emailAddresses:
```

I wonder if we need a prefix here, it belongs to `notification` object.

---

Thanks for the feedback! It's currently already implemented with `notificationEmailAddresses`, we're working on a follow up RFC where we will change this as well.